distribution/docker/knotx/conf/openapi.yaml

```diff
@@ -0,0 +1,50 @@
+openapi: "3.0.0"
+info:
+  version: 1.0.0
+  title: Knot.x Starter Kit
```

Comment:
I would change it to something like: `Knot.x Stack with Docker example`.
The same with a description.

Reply:
✔️ done